src/AddTodo/AddTodo.js

```diff
@@ -0,0 +1,30 @@
+
+const AddTodo = ({item, setItem, handleAddTask}) => {
+
+    const handleTaskChange = (e)=> {
```

Comment:
nit: Can we make sure the spacing is consistent throughout the code? Please configure some basic eslint with your project.
```suggestion
    const handleTaskChange = (e) => {
```